example/r3fDemo.jsx

```diff
@@ -0,0 +1,286 @@
+    /> 
+  </>
+}
+
+function SuziModel(props) {
+  const { nodes } = useGLTF('https://vazxmixjsiawhamofees.supabase.co/storage/v1/object/public/models/suzanne-high-poly/model.gltf')
+  return (
+    <mesh castShadow receiveShadow geometry={nodes.Suzanne.geometry} {...props}>
+      <meshStandardMaterial color={"#9d4b4b"} {...props.materialProps} />
+    </mesh>
+  )
+}
+
+function Additional3dObjects({commonPluginsProps}) {
+  const {lat, lon, fade, debug} = commonPluginsProps
+  return <>
+    {/* Other standard r3f objects added to scene */}
+    {/* either to local frame */}
+    <SuziModel position={[0, 0, 0]} />
+
+    {/* or ENUFrame center to provided lat-lon-height */}
+    <EastNorthUpFrame
+      lat = {lat * Math.PI / 180}
+      lon = {lon * Math.PI / 180}
+      height = { 100}
+      az = {0}
+      el = {0}
+      roll = {0}
+    > 
+      <SuziModel position={[0, 0, 2]} rotation-z={Math.PI/2 * 0} rotation-y={- Math.PI/2} scale={1} materialProps={{color:'#0000cc'}} />
+    </EastNorthUpFrame>
+  </>
+}
+
+function CommonPlugins ( props ) {
+  return <>
+    <TilesPlugin plugin={ GLTFExtensionsPlugin } 
+      dracoLoader={dracoLoader}
+      ktxLoader={ktx2Loader}
+      autoDispose={false}
+      // both args and props/options do work to pass loaders
+      // args = {{
+      //   dracoLoader, ktxLoader:ktx2Loader
+      // }}
+    />
+    {
+      (props.lat && props.lon) ? 
+      <TilesPlugin plugin={ ReorientationPlugin } 
+        lat={props.lat * Math.PI / 180}
+        lon={props.lon * Math.PI / 180}
+        height={props.height || 100}
+        up={'+z'}
+        recenter={true}
+        key={`${props.lat}-${props.lon}-${props.height}`}
+      /> : 
+      // If no lat/lon passed as props, recenter automatically
+      <TilesPlugin plugin={ ReorientationPlugin } 
+        recenter={true}
+      />  
+    }
+    {props.fade && <TilesPlugin plugin={ TilesFadePlugin } fadeDuration={props.fadeDuration || 500} />}
+    {props.debug && <TilesPlugin plugin={ DebugTilesPlugin } 
+      colorMode={NONE} // NONE, SCREEN_ERROR, GEOMETRIC_ERROR, DISTANCE, DEPTH, RELATIVE_DEPTH, IS_LEAF, RANDOM_COLOR, RANDOM_NODE_COLOR, CUSTOM_COLOR, LOAD_ORDER
+      displayBoxBounds={true}
+      displayRegionBounds={false}
+    />}
+    <TilesPlugin plugin={ TileCompressionPlugin } 
+      generateNormals={false}
+      disableMipmaps={true}
+      compressIndex={false}
+      // compressNormals={true} normalType={Int8Array}
+      // compressUvs={false} uvType={Int8Array}
+      // compressPosition={false} positionType={Int16Array}
+    />
+  </>
+}
+
+const URL = 'https://raw.githubusercontent.com/NASA-AMMOS/3DTilesSampleData/master/msl-dingo-gap/0528_0260184_to_s64o256_colorize/0528_0260184_to_s64o256_colorize/0528_0260184_to_s64o256_colorize_tileset.json';
+
+const googleApiKey_ = import.meta.env.VITE_IONACCESSTOKEN || 'put-your-api-key-here' 
+const ionAccessToken_ = import.meta.env.VITE_IONACCESSTOKEN || 'put-your-api-key-here'
```

Comment:
I'd prefer to use local storage for storing these fields, as is done [in the other examples](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/example/ionExample.js#L18). Setting env variables is fairly opaque even for development, I think.

Reply:
Leva has no [persistence](https://github.com/pmndrs/leva/issues/113) to localStorage yet - multiple threads on global store/state integration listed [here](https://github.com/pmndrs/leva/issues/295) - but I can use onChange to persist individual values to localStorage, and setStore initially. Will try that right now.

Reply:
Alright, just added localStorage support for google and cesium api keys! Note if we want automatic localStorage support with every leva controls (which is not part of this pr), I've also added a snippet [there](https://github.com/pmndrs/leva/issues/295#issuecomment-2404777836)

Reply:
That's okay - the local storage saving is mostly useful for the access tokens for viewing the data.